src/TestIds.ts

```diff
@@ -20,6 +20,7 @@ import { Platform } from 'react-native';
 export const TestIds = {
   APP_OPEN: '',
   BANNER: '',
+  CUSTOM_TARGETING: {},
```

Comment:
TestIds serves as an object that holds al the Test Ad Unit ID's -> https://developers.google.com/admob/ios/test-ads
Whereas customTargeting is a request option.